tests/testthat/test-new-test.R

```diff
@@ -137,3 +137,27 @@ test_that("seed output check", {
   expect_identical(seed_analyze_uu$address.msg,"Uninitialised value was created by a stack allocation")
 })
 }
+
+
+test_that("When an error occurs before the test harness execution, it exits with an error message. ", {
+  
+  fun_roob <- file.path(path,"inst","testfiles","rcpp_read_out_of_bound") 
+  exe_roob <- "rcpp_read_out_of_bound_DeepState_TestHarness"
+  
+  if(!dir.exists(fun_roob)){
+    deepstate_fun_create(path, basename(fun_roob))
+  }
+
```

Comment:
Inside of a test_that block I prefer to not put any blank lines, so then the entire block is easily executable, for example in Emacs/ESS C-c C-c executes all code in the current block (from the previous blank line, up to the next blank line).

Reply:
@tdhock, thank you for informing me about this. With commit 5c67437 I removed these empty lines from the new test as well as previous tests I wrote.